test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -370,18 +374,71 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			})
 		})
 	})
+
+	var _ = ginkgo.Context("Requeuing timestamp set to Creation", func() {
+
+		ginkgo.BeforeEach(func() {
+			podsReadyTimeout = 3 * time.Second
+			requeuingTimestamp = config.CreationTimestamp
+		})
```

Comment:
there  should be a corresponding `AfterEach` resetting the original values.

Reply:
Ideally, yes, but we can leave it for a follow up.

Just add an AfterEach for this new case in this PR.

Reply:
Gotcha, I guess it makes sense to add some variables for tracking the original values?

Reply:
Take a look at [this commit](https://github.com/kubernetes-sigs/kueue/pull/1542/commits/edc8d7d552bc774d57fb6359cbac9ef58761412d) to see if that is what you prefer.